test/integration/graphql/business.test.js

```diff
@@ -70,8 +94,84 @@ describe('Query.customer', () => {
 
     expect(result).toEqual({
       data: {
-        business: JSON.parse(JSON.stringify(transformedOrganisation))
+        business: JSON.parse(JSON.stringify({
+          ...transformedOrganisation,
+          ...transformedOrganisationCSApplications
+        }))
       }
     })
   })
 })
+
+describe('Query businessApplications', async () => {
+  const result = await graphql({
+    source: `#graphql
+    query BusinessApplications {
+        businessApplications(id: "5444918") {
+            applicationStatus {
+                id
+                open
+                status
+                type
+                sector
+                year
+                FRN
+                office
+            }
+            csClaims {
+                schemeYear
+                type
+                status
+                lastMovement
+            }
+        }
+    }
+    `,
+    variableValues: {
+      customerId: '5444918'
+    },
+    schema,
+    contextValue: context()
```

Comment:
Either I will need to update this or you will depending on my auth PR to use fakeContext